alluxiofs/core.py

```diff
@@ -46,49 +46,27 @@ class AlluxioFileSystem(AbstractFileSystem):
 
     def __init__(
         self,
-        etcd_hosts=None,
-        worker_hosts=None,
-        options=None,
-        concurrency=64,
-        etcd_port=2379,
-        worker_http_port=28080,
-        preload_path=None,
         target_protocol=None,
         target_options=None,
         fs=None,
+        alluxio_client=None,
         test_options=None,
         **kwargs,
     ):
         """
         Initializes an Alluxio filesystem on top of underlying filesystem
         to leveraging the data caching and management features of Alluxio.
 
-        The Alluxio args:
-            etcd_hosts (str, optional): A comma-separated list of ETCD server hosts in the format "host1:port1,host2:port2,...".
```

Comment:
Should we keep the alluxiofs doc? this is because s3fs, although the parameters are passed in **kwargs but still in the doc, user may view this doc directly for supported parameters? but it will be hard to maintain as well 

@jja725 what are your thoughts?

Reply:
yeah agree let's keep the doc here otherwise no one can find the proper options to use